pages/mesosphere/dcos/1.14/gui/nodes/index.md

```diff
@@ -8,26 +8,129 @@ render: mustache
 
 ![Nodes](/mesosphere/dcos/1.14/img/nodes-masters-ee-dcos-1-12.png)
 
-Figure 3 - Masters Nodes tab
+Figure 4 - Masters tab
+
+
+# Nodes instance
+
+Clicking on a **Host IP** opens a Nodes instance page, which provides more information about the node. You can use the Search box or the drop down filter to narrow your list. The **Nodes** instance page has three tabs: [Tasks](#tasks), [Health](#health), and [Details](#details). The **Tasks** tab is displayed by default.
+
+## Tasks
+
+![Tasks tab](/mesosphere/dcos/1.14/img/GUI-Nodes-Tasks-Tab.png)
+
+Figure 5 - Tasks tab
+
+The **Tasks** tab displays the following information:
+
+
+| Name | Description |
+|----------|-----------|
+| ID | Unique identifier for this instance of a service. Consists of the name of the service (for example, `confluent-kafka` plus configuration details.)  |
+| Name |  Unique name of this instance of a service. Consists of a series of words separated by dashes (for example, `data-science-engine`). Each word must be at least 1 alphanumeric character and may only contain digits (`0-9`), dashes (`-`), dots (`.`), and lowercase letters (`a-z`). The word may not begin or end with a dash.     |
+| Host |  IP address of this node.     |
+| Zone |   Cloud provider zone.    |
+| Region | Cloud provider region. If left undefined this will run in your local region. |
+| Status |  The current state of the node.     |
+| Health | The current health of the node.      |
+| Logs | Clicking on the "View Logs" icon will open the logs for this instance of the service.  |
+| CPU |  The number of CPU shares allocated to the process.     |
+| Mem |  The amount of memory, in MB, allocated to the process.    |
+| GPU |   Number of GPUs available to this node.    |
+| Updated | Time, in minutes, since last update. |
+
+A vertical line of dots in the upper right corner opens a menu which allows you to drain or deactivate a node. 
+
+![Deactivation menu](/mesosphere/dcos/1.14/img/GUI-Nodes-Drain-Menu.png)
+
+Figure 6 - Deactivation menu
+
+For more information on draining nodes, see [Draining a Node](/mesosphere/dcos/1.14/administering-clusters/draining-a-node/). For more information on deactivating a node, see the documentation on how to [Shut Down and Decomission Nodes](/mesosphere/dcos/1.14/administering-clusters/delete-node/).
+
+## Health
+
+The **Health** tab displays the status of your node's health checks.
+
+![Health tab](/mesosphere/dcos/1.14/img/GUI-Nodes-Health-Tab.png)
+
+Figure 6 - **Nodes > Health** tab
+
+You can filter this list using the **All Health Checks** drop down menu.
+
+![All health checks menu](/mesosphere/dcos/1.14/img/GUI-Nodes-Health-Filter.png)
+
+Figure 7 - Health checks filter
+
+## Details
+
+The **Nodes > Details** tab will show you the identification information for this node, as well as its Status and Resources.
+
+![Details panel](/mesosphere/dcos/1.14/img/GUI-Nodes-Details-Tab.png)
+
+Figure 8 - Nodes details tab
+
+### Service details
```

Comment:
This tasks table is in fact the same you get clicking on a service name from the Services Overview page. We could simply link that page (if it exists) to avoid duplicates.